src/components/MovieList/MovieList.jsx

```diff
@@ -1,19 +1,28 @@
 /* eslint-disable react/jsx-key */
 import React, { useEffect } from "react";
-import { useSelector, useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
+import PropTypes from "prop-types";
 import Film from "../Film/Film";
-import { getListFilms, getGenre } from "../../modules/selectors/index";
-import requestFilms from "../../modules/actions/requestFilms";
+import requestFilmsByGenre from "../../modules/actions/requestFilmsByGenre";
+import requesFilms from "../../modules/actions/requestFilms";
+import { useParams } from "react-router";
+import Loading from "../Loading/Loading";
 
 import "./style.scss";
 
-const MovieList = () => {
-  const films = useSelector(state => getListFilms(state));
-  const genre = useSelector(state => getGenre(state));
+const MovieList = ({ films, genre, movieListView }) => {
```

Comment:
probably we can take `films` from the redux?
and avoid dropping them from the props